package.json

```diff
@@ -46,15 +46,19 @@
     "build:refactor:fspiop:v1.1": "npx ts-node --transpile-only src/scripts/openapi-refactor.ts docs/fspiop-rest-v1.1-openapi3-snippets.yaml docs/fspiop-rest-v1.1-openapi3-snippets.yaml",
     "build:refactor:fspiop:v2.0": "npx ts-node --transpile-only src/scripts/openapi-refactor.ts docs/fspiop-rest-v2.0-openapi3-snippets.yaml docs/fspiop-rest-v2.0-openapi3-snippets.yaml",
     "build:refactor:fspiop:v2.0_ISO20022": "npx ts-node --transpile-only src/scripts/openapi-refactor.ts docs/fspiop-rest-v2.0-ISO20022-openapi3-snippets.yaml docs/fspiop-rest-v2.0-ISO20022-openapi3-snippets.yaml",
-    "lint": "eslint --ext ts --ext js test src",
-    "lint:fix": "eslint --fix --ext ts --ext js test src",
+    "lint": "npm run build && eslint --ext ts --ext js test src",
+    "lint:fix": "npm run build && eslint --fix --ext ts --ext js test src",
     "prepare": "husky install && npm run build",
     "dep:check": "ncu -e 2",
     "dep:update": "ncu -u",
     "release": "standard-version --releaseCommitMessageFormat 'chore(release): {{currentTag}} [skip ci]'",
     "snapshot": "standard-version --no-verify --skip.changelog --prerelease snapshot --releaseCommitMessageFormat 'chore(snapshot): {{currentTag}}'",
     "test": "npm run test:unit",
     "test:unit": "jest --testMatch '**/test/dto/*.(test|spec).ts'",
+    "test:xunit": "echo 'No xunit tests'",
```

Comment:
I think the orb runs this `test:xunit` and not `test:unit` so in the command you can just run add `npm -s run test:unit` otherwise it won't run the unit tests.